release-notes/opensearch-release-notes-2.6.0.md

```diff
@@ -0,0 +1,539 @@
+# OpenSearch and OpenSearch Dashboards 2.6.0 Release Notes
+
+## Release Highlights
+
+### New Features
+
+
+### Experimental Features
+
```

Comment:
* A new model health dashboard lets you view the location and status of machine learning models, simplifying management of semantic search and other ML workloads.
* This release adds support for AWS Signature Version 4 (AWS SigV4) as a request authentication method for the experimental multiple data sources feature, which allows you to add multiple data sources to a single dashboard.